lib/experience-bootstrap.js

```diff
@@ -0,0 +1,30 @@
+const {
+  loadConfig,
+  logResult,
+  logError
+} = require('./utils');
+const getApi = require('./get-api');
+const { experience } = require('./get-download-params');
+const getDownloader = require('./get-downloader');
+const downloader = getDownloader(experience);
+const { pipe, propOr, find, propEq } = require('omnibelt');
+const hasBootstrapped = pipe(propOr([], 'ftueTracking'), find(propEq('name', 'experience')));
```

Comment:
should only be considered "bootstrapped" if `version >= 2` - see equivalent front end check (in `shouldShowModal`)